fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AstBuilderTest.java

```diff
@@ -92,4 +94,66 @@ public void testTruncatePartition() throws Exception {
         TruncatePartitionClause c = (TruncatePartitionClause) alterClauses.get(0);
         Assert.assertTrue(c.getPartitionNames().getPartitionNames().get(0).equals("p1"));
     }
+
+    @Test
```

Comment:
Except UT for analyzer, plz add more UT for execution, you can refer to `fe/fe-core/src/test/java/com/starrocks/catalog/DropPartitionTest.java`.

Reply:
We have a 80% coverage requirement.

Reply:
Alright, I will add unit tests.